Python.md

```diff
@@ -1,6 +1,220 @@
+You may find yourself pondering over which tool is best suited for developing automation workflows leveraging extensive DNA Center API capabilities. Based on NetDevOps Survey conducted in 2020, the following is the breakdown of tools used in the field for automating the generation and/or the deployment of configurations at scale, [NetDevops 2020 Config Generation Tools Breakdown](results/2020/netdevops_survey_2020_config-gen-deploy_tool.png). By far, one of the most popular Automation platforms cited is RedHat Ansible. Given its popularity, it may be conducive to consider Ansible and its very mature Cisco DNA Center Module Galaxy collection as a starting point, [Redhat Ansible DNA Center Galaxy Collection](https://galaxy.ansible.com/cisco/dnac). At its core, Ansible DNA Center module collection is a set of Python directives, abstracting DNA Center native REST API and integrating those into Ansible constructs. As such, one of the biggest benefits of leveraging Ansible DNA Center Galaxy collection is standardization and abstraction from underlying REST API implementation details. This approach works great in organizations that od not have dedicated developer teams who are tasked with maintaining Python code responsible for DNA Center Integrations. However, Ansible approach brings its own set of challenges when it comes to building anything more than just linear, sequential workflows. Playbooks containing even moderately complex business logic become unwieldy and start growing in complexity exponentially. Another aspect to consider in same context is large datasets which DNA Center API may return in some of the larger deployments, and the requirement to 'parse' or 'tokenize' that data if it were two be operated on through Ansible. 
+This is where it becomes more practical to consider Python for developing such workflows. Python offers the utmost flexibility and elegance of developing your own applications or integrations with DNA Center. Python's dependency on DNA Center version-specific REST API implementation is further abstracted by Cisco maintained DNA Center Python SDK. 
+
+DNA Center programmable REST API capabilities are covered in our previous tutorial section, [REST API](./RestAPI.md) - REST API and Cisco DNA Center. This section will build upon API foundational concepts introduced in that earlier section
+
+In this section we will cover Python integrations with Cisco DNA Center's "Business and Network Intent APIs".
+
+Before diving into developing your own Python automation scripts with DNA Center, we will work through initial steps required to setup your development environment by installing:
+- Python environment 
+- Python PIP module manager
+- Cisco DNA Center Python SDK
+
+## Prerequisites
+* It is assumed that the reader has general level familiarity with Python language before proceeding with examples outlined in this section. 
+* Please ensure that you have Python v3.x installed on your machine 
+
```

Comment:
In this section and following sections explain the whys of whats going on please...